notifications/bridge.go

```diff
@@ -23,8 +24,8 @@ var _ Listener = &bridge{}
 
 // URLBuilder defines a subset of url builder to be used by the event listener.
 type URLBuilder interface {
-	BuildManifestURL(name, tag string) (string, error)
-	BuildBlobURL(name string, dgst digest.Digest) (string, error)
+	BuildManifestURL(name reference.Named, tag string) (string, error)
```

Comment:
It should take `reference.NamedTagged`, and `BuildBlobURL` should take `reference.Canonical`. I'll make these changes.